src/blockchain.js

```diff
@@ -156,10 +156,13 @@ Blockchain.doConnect = function(connectionList, opts, doneCb) {
           self.blockchainConnector.setDefaultAccount(accounts[0]);
         }
 
-        const connectionError = new BlockchainConnectionError(connectionErrs);
+        let connectionError;
```

Comment:
Is there a possibility of `err` being populated (from `self.blockchainConnector.getAccounts`? Could handle with:
```suggestion
        let connectionError;
        if(err) {
          cb(err);
          return doneCb(err);
        }
```